weaviate/gql/get.py

```diff
@@ -83,7 +84,13 @@ def __init__(
         if not isinstance(class_name, str):
             raise TypeError(f"class name must be of type str but was {type(class_name)}")
         if properties is None:
-            properties = []
+            class_schema = schema.get(class_name=class_name)
+            if class_schema:
```

Comment:
What exactly is the ìf` for?
Afaik get returns an exception if the class does not exists

Reply:
Removed it.